### PR TITLE
updated according to rubric

### PR DESCRIPTION
**Pull request by majesticio**
adapter now uses parseFloat, let, luxon and got

### CODE REVIEW

File: src/adapters/au_act.js [hunk 1]
```diff
@@ -1,66 +1,71 @@
...
-      return cb({message: 'Failure to load data url.'});
-    }
-    // Wrap everything in a try/catch in case something goes wrong
+  }).then(res => {
+    const body = res.body;
+    
     try {
-      // Format the data
-      var data = formatData(JSON.parse(body), source);
+      let data = formatData(JSON.parse(body), source);
```
Comment (russbiggs) [inline src/adapters/au_act.js:26]: this can be `const` its never changed

File: src/adapters/au_act.js [hunk 2]
```diff
@@ -1,66 +1,71 @@
...
+  let types = {
     'no2': 'no2',
     'o3_1hr': 'o3',
     'co': 'co',
     'pm10': 'pm10',
     'pm2_5': 'pm25'
   };
 
-  var units = {
+  let units = {
```
Comment (russbiggs) [inline src/adapters/au_act.js:60]: this should be `const` its never reassigned

File: src/adapters/au_act.js [hunk 3]
```diff
@@ -1,66 +1,71 @@
...
+  
+}
 
-var formatData = function (data, source) {
-  var parseDate = function (string) {
-    var date = moment.tz(string, 'Australia/Sydney');
-    return {utc: date.toDate(), local: date.format()};
+
+let formatData = function (data, source) {
+  let parseDate = function (string) {
```
Comment (russbiggs) [inline src/adapters/au_act.js:43]: this should be `const` its never reassigned

**Review by russbiggs — CHANGES_REQUESTED** [review]

**Comment by majesticio** [comment]
merged into update-libraries

**Pull request closed without merging**